### ISSUE TITLE
Guys, are you planning a google colab laptop for this? Very necessary

### ISSUE THREAD

**Issue by petrusinov** [eliahuhorwitz/DeepSIM#8]

**Comment by eliahuhorwitz** [comment]
Indeed, in the coming weeks we will publish one as well as a few pretrained models

**Comment by petrusinov** [comment]
thanks

**Comment by Lime-Cakes** [comment]
I've had some success running this on google colab, by just git cloning it and running with gpu. I can share the notebook, but it's better to wait for the author's official implementation.

**Comment by petrusinov** [comment]
> I've had some success running this on google colab, by just git cloning it and running with gpu. I can share the notebook, but it's better to wait for the author's official implementation.

If possible, share your work for now. It is not known when the author will upload his google collab

**Comment by Lime-Cakes** [comment]
> > I've had some success running this on google colab, by just git cloning it and running with gpu. I can share the notebook, but it's better to wait for the author's official implementation.
> 
> If possible, share your work for now. It is not known when the author will upload his google collab

https://colab.research.google.com/drive/11ThquWQ63SSUbEEZe3r1KLdytAHWPbGu?usp=sharing

I haven't finished training yet. Seems like colab is low on GPU lately (or google just hates me). It's possible that I messed something up, since an epoch takes around 1.5 second.

**Comment by rickkk856** [comment]
```
End of epoch 1007 / 16000 	 Time Taken: 1.8693 sec
End of epoch 1008 / 16000 	 Time Taken: 1.8707 sec
End of epoch 1009 / 16000 	 Time Taken: 1.8733 sec
End of epoch 1010 / 16000 	 Time Taken: 1.8709 sec
End of epoch 1011 / 16000 	 Time Taken: 1.8680 sec
End of epoch 1012 / 16000 	 Time Taken: 1.8709 sec

```

Same here...
